packages/components/src/components/tab-panel/tab-panel.stories.ts

```diff
@@ -36,7 +53,23 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
```

Comment:
Same as above. No need to add name, only in screenshot tests

Reply:
Updated.